scanpy/external/tl/_harmony_timeseries.py

```diff
@@ -0,0 +1,213 @@
+"""Harmony time series for data visualization with augmented affinity matrix at
+    discrete time points
+"""
+
+import pandas as pd
+from anndata import AnnData
+
+from ... import logging as logg
+
+
+def harmony_timeseries(data: AnnData, tp: str):
+    """Harmony time series for data visualization with augmented affinity matrix
+    at discrete time points [Nowotschin18i]_.
+
+    Harmony time series is a framework for data visualization, trajectory
+    detection and interpretation for scRNA-seq data measured at discrete
+    time points. Harmony constructs an augmented affinity matrix by augmenting
+    the kNN graph affinity matrix with mutually nearest neighbors between
+    successive time points. This augmented affinity matrix forms the basis for
+    generated a force directed layout for visualization and also serves as input
+    for computing the diffusion operator which can be used for trajectory
+    detection using **Palantir**.
```

Comment:
Why are you not using proper links?

```suggestion
    detection using Palantir_.

    .. _Palantir: https://github.com/dpeerlab/Palantir
```